Frontend/src/components/listUpdateProfileFields.js

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {ScrollView, View, Text, TextInput, StyleSheet} from 'react-native';
+
+import UpdateProfileField from '../components/updateProfileField';
+
+const ListUpdateProfileField = ({
+  userReducer,
```

Comment:
Would rename to userData or just user because data might come from other sources as well in case this component will be reused